consul/src/main/java/com/linecorp/armeria/internal/consul/CatalogClient.java

```diff
@@ -53,6 +55,9 @@ static CatalogClient of(ConsulClient consulClient) {
         return new CatalogClient(consulClient);
     }
 
+    private static final String DATACENTER_PARAM = "dc";
+    private static final String FILTER_PARAM = "filter";
+
```

Comment:
Please move this static field above `static` methods.
https://armeria.dev/community/developer-guide/#organize